frontend/contexts/UserContext.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */

```

Comment:
If its unused just remove it?

Reply:
it's used we are just setting function headers and eslint dumb